backend/src/database/users.go

```diff
@@ -311,37 +312,30 @@ func (db *DB) GetNumberOfActiveUsersForTimePeriod(args *models.QueryContext, act
 	if facilityId != nil {
 		tx = tx.Where("facility_id = ?", *facilityId)
 	}
+	tx = tx.Where("role = 'student'")
 	if err := tx.Count(&count).Error; err != nil {
-		return 0, newGetRecordsDBError(err, "users")
+		return 0, days, newGetRecordsDBError(err, "users")
+	}
+	if days == 0 {
+		days = 1
 	}
-	return count, nil
+	return count, days, nil
 }
 
-func (db *DB) NewUsersInTimePeriod(args *models.QueryContext, days int, facilityId *uint) (int64, int64, error) {
-	var admin_count int64
+func (db *DB) NewUsersInTimePeriod(args *models.QueryContext, days int, facilityId *uint) (int64, error) {
 	var resident_count int64
 	daysAgo := time.Now().AddDate(0, 0, -days)
-	tx := db.WithContext(args.Ctx).Model(&models.User{}).Where("role ='student'")
+	tx := db.WithContext(args.Ctx).Model(&models.User{}).Where("role = 'student'")
 	if days != -1 {
 		tx = tx.Where("created_at >= ?", daysAgo)
 	}
 	if facilityId != nil {
 		tx = tx.Where("facility_id = ?", *facilityId)
 	}
 	if err := tx.Count(&resident_count).Error; err != nil {
-		return 0, 0, newGetRecordsDBError(err, "users")
-	}
-	tx = db.WithContext(args.Ctx).Model(&models.User{}).Where("role IN ?", models.AdminRoles)
-	if days != -1 {
-		tx = tx.Where("created_at >= ?", daysAgo)
-	}
-	if facilityId != nil {
-		tx = tx.Where("facility_id = ?", *facilityId)
-	}
-	if err := tx.Count(&admin_count).Error; err != nil {
-		return 0, 0, newGetRecordsDBError(err, "users")
+		return 0, newGetRecordsDBError(err, "users")
 	}
-	return resident_count, admin_count, nil
+	return resident_count, nil
 }
 
 func (db *DB) GetTotalLogins(args *models.QueryContext, days int, facilityId *uint) (int64, error) {
```

Comment:
Can you add a check here? I was getting an error if I loaded this page and there weren't any logins from in the `login_activity` table yet. It is possible an Admin might try to access the page before any Residents are added into the system.

```
server-1            | {"handler":"handleLoginMetrics","level":"error","method":"GET","msg":"Error occurred is sql: Scan error on column index 0, name \"sum\": converting NULL to int64 is unsupported","path":"/api/login-metrics","time":"2025-05-19T15:36:15Z"}{"handler":"handleWebsocketConnection","level":"debug","method":"GET","msg":"WebSocket connection established","path":"/api/ws/listen","time":"2025-05-19T15:36:15Z"}
```